packages/api/src/store/db.ts

```diff
@@ -195,36 +205,44 @@ export class DB {
     query: string | QueryConfig<I>,
     values?: I
   ): Promise<QueryResult<T>> {
-    let queryLog;
+    let queryLog: string;
+    const labels: any = {};
     if (typeof query === "string") {
       queryLog = JSON.stringify({ query: query.trim(), values });
+      labels.query = query.trim();
     } else {
+      labels.query = query.text;
       queryLog = JSON.stringify(query);
     }
-    let result;
+    let result: QueryResult;
     logger.info(`runQuery phase=start query=${queryLog}`);
     const start = Date.now();
+    const queryTimer = this.metricHistogram.startTimer(labels);
     try {
       result = await pool.query(query, values);
+      labels.phase = "success";
```

Comment:
I think the term phase made sense when we also had a `start` phase. Since we are only splitting here whether the query was a success or error, WDYT of calling it `result` instead?

Reply:
my first name chosen was `success = true/false`, but just kept it as `phase` for consistency. otoh, `result` label felt like it should record the count of rows from query result/equivalent and not the `status` (?) of the query itself.